src/Lecture1.hs

```diff
@@ -108,7 +130,11 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: [Char]->Int
+strSum str = sum l
+    where 
+        c = words str
+        l = map (\x -> read x :: Int) c
```

Comment:
As you specify the top-level type signature for this function, the compiler would be able to figure out what type `read` function has in this case due to type inference, so you can skip specifying it in the body:

```suggestion
        l = map read c
```

If you do this, you can notice that it's possible to implement `strSum` a simple one-liner 🙂

Reply:
Right, got it. Thanks!